### ISSUE TITLE
🚀 FEAT: Add contracts from remapping.txt

### ISSUE THREAD

**Issue by blueh4mster** [BlocSoc-iitr/Beskar#2]
🚀 Feature

Description: Currently, the tool only has the functionality for mutating and testing all the smart contracts in the `\src` directory. Implement a feature such that contracts can be mutated according to the paths in the `remappings.txt` file. 

Objective: Include the mutations and testing function for contracts in accordance with the remappings.

**Comment by x-senpai-x** [comment]
Please assign this

**Comment by x-senpai-x** [comment]
remappings is assumed to be located at ./remappings.txt right ?

**Comment by Oyase-shinobi** [comment]
Hello @blueh4mster 
I would love to engage in your project

**Comment by blueh4mster** [comment]
> remappings is assumed to be located at ./remappings.txt right ?

Yes, the remappings file is to be included in the foundry project. If not present, the tool does nothing.

**Comment by Oyase-shinobi** [comment]
Hi @blueh4mster 
Thank you for reaching out me
Could you share your discord ID for better communication?

**Comment by x-senpai-x** [comment]
@blueh4mster we need to have these rust files and the solidity files in the same src directory?

**Comment by x-senpai-x** [comment]
When i use  cargo run -- run 
  Rust files are being overwritten by the Solidity files during the execution
